### ISSUE TITLE
Make the .meta file format identical with reST metadata

### ISSUE THREAD

**Issue by Kwpolska** [getnikola/nikola#954]
As discussed in #951 and [on the mailing list](https://groups.google.com/forum/#!topic/nikola-discuss/mwo4t7phStA).

> Right now, you can just put seven meta elements there: title, slug, date, tags, links, description, and type. (type is new and kinda useless by default) There is no support for any of the other metadata fields, like nocomments, template, password, category (!), (no)annotations, author, or hidetitle. We plan to make it possible by just using the same meta format as we do in reST posts now — stick ".. NAME: " in front of everything, and be explicit.
> 
> Note that the 2-file-format is not going away (as it’s required by ipynb, creolewiki, pandoc/latex and possibly others).  And there will be a nice converter to fix everything for you.
> 
> **tl;dr:** The .meta file = header of our reST posts.

For the _provide a script to upgrade_ part, there are two ways:

1. Provide a standalone script, like [this one-liner I wrote in July](https://groups.google.com/forum/#!topic/nikola-discuss/07otrEqbXao):

``` bash
awk 'BEGIN {RS = ""; FS="\n"}; {print ".. title: "$1"\n.. slug: "$2"\n.. date: "$3"\n.. tags: "$4"\n.. link: "$5"\n.. description: "$6"\n"}'

```

Usage:

``` bash
for i in *.meta; do
    cat $i | awk 'BEGIN {RS = ""; FS="\n"}; {print ".. title: "$1"\n.. slug: "$2"\n.. date: "$3"\n.. tags: "$4"\n.. link: "$5"\n.. description: "$6"\n"}' > $i'2'
done
rename .meta2 .meta *.meta2
```

2. Write a full-blown script that automates everything and is a Nikola plugin, making usage even easier, i.e.

``` console
$ nikola install_plugin meta_upgrade
[2014-02-01T15:35:59Z] NOTICE: install_plugin: Downloading: http://plugins.getnikola.com/v7/meta_upgrade.zip
[2014-02-01T15:36:00Z] NOTICE: install_plugin: Extracting: meta_upgrade into plugins
$ nikola meta_upgrade
Finding .meta files... 500 found
(500/500) Upgrading meta files...
Done.
$ rm -rf plugins/meta_upgrade # YAGNI
```
